docs/reference/db/authorization/rules.md

```diff
@@ -0,0 +1,209 @@
+# Rules
+
+<!-- TODO: Add examples that explicitly show how roles work in the context of rules -->
+
+## Introduction
+
+<!-- TODO: Rewrite this: What rules are and what they do -->
+
+Every rule must specify:
+
+- `role` (required) — A role name. It's a string and must match with the role(s) set by an external authentication service.
+- `entity` (required) — The Platformatic DB entity to apply this rule to.
+- `defaults` (optional) — Configure entity fields that will be
+  [automatically set from user data](#set-entity-fields-from-user-metadata).
+- One entry for each supported CRUD operation: `find`, `save`, `delete`
+
+## Operation checks
+
+Every entity operation can specify `checks` used for the authorizations.
```

Comment:
What is an `entity operation`? It;'s better to refresh this refers to both GraphQL and REST endpoints as this instruments `sql-mapper`

Reply:
I've reworded this sentence as:

> Every entity operation — such as `find`, `insert`, `save` or `delete` — can have authorization `checks` specified for them.

I've also added a sentence to the beginning of the introduction for this page that says:

> Authorization rules can be defined to control what operations users are able to execute via the REST or GraphQL APIs that are exposed by a Platformatic DB app.